packages/turf-shortest-path/lib/javascript-astar.d.ts

```diff
@@ -0,0 +1,36 @@
+// Copied from @types/javascript-astar@0.0.34
```

Comment:
We can probably just update definitely-typed directly, that's usually pretty quick to do.
Otherwise we probably need to preserve whatever license blob is relevant from upstream.

Reply:
If you're happy to, I'll add a license blob as a temporary measure so this PR can proceed. In the background will see what can be done to fix the types at Definitely Typed and tidy up down the track.

Reply:
I'm not a lawyer but that seems fine. Also this being A* explains why the pathing looks weird, I bet Dijkstra's would be slower but give a more intuitive looking answer.

Reply:
Included the license text as a workaround for now.